### DISCUSSION TITLE
QR Code in URL Bar

### DISCUSSION THREAD

**Discussion by jgonzales20** [zen-browser/desktop#1341] [Ideas]
Integrate this extension into the default Zen Browser experience.

https://addons.mozilla.org/en-US/firefox/addon/qr-code-address-bar/

**Comment by deltaxsingh** [comment]
Damn had no idea this even existed. Honestly better implementation than Chrome. Thanks!

**Comment by XJY00** [comment]
+1
This is an extremely convenient and important feature
